### DISCUSSION TITLE
Week 4

### DISCUSSION THREAD

**Discussion by munoztd0** [munoztd0/DAP_2021#7] [Announcements]
Hello @lucilefavero @ap1987-ap @faureunige @danielsgranja ,

I hope you add a good weekend.

We will meet as usual this Monday after class at 4pm to discuss new information as well as to advance on the modeling.

Things to discuss:

"may be simpler to schedule the orals with your group: please 
discuss with them to identify 3 half-days suitable in the week from 7 to 
11 February and let me know so we can make a calendar
   - the students can use more advanced methods than those presented in 
the course but should also use simpler methods discussed in the course 
and highlight their differences. In the orals they can be asked details 
about the methods presented in the course but not about the advanced 
methods (that's why it is important an analysis with simpler methods is 
included in the report"

**Comment by munoztd0** [comment]
Hello @lucilefavero and @ap1987-ap ,
Since the others cannot be there today I was wondering if any other would work for the 4 of you. What about Wednesday afternoon or Friday afternoon @danielsgranja @faureunige ?

Comment below 👇 please.

**Reply by faureunige** [reply]
Hi, sorry all ! i am free on wednesday afternoon and friday afternoon. Thanks

**Comment by danielsgranja** [comment]
Wednesday after 16h and friday afternoon 14h-18h i have class :/

**Comment by ap1987-ap** [comment]
Hi, but you would still come today for me and Lucille?ThanksAnjaOn 6 Dec 2021 15:03, David Muñoz Tord ***@***.***> wrote:

Hello @lucilefavero and @ap1987-ap ,

Since the others cannot be there today I was wondering if any other would work for the 4 of you. What about Wednesday afternoon or Friday afternoon @danielsgranja @faureunige ?

Comment below 👇 please.

**Reply by ap1987-ap** [reply]
Because I actually traveld 2h today to come here for that ;-)

**Comment by munoztd0** [comment]
Well that is the question. If you still prefer doing it today I will come without any issue.

**Reply by ap1987-ap** [reply]
So yes, i think that would be great if you could come today for me and lucille.
Thanks

**Reply by munoztd0** [reply]
Perfect, then no worries we do as usual!

**Reply by ap1987-ap** [reply]
Perfect, thank you

**Comment by munoztd0** [comment]
For @faureunige and @danielsgranja let's meet via zoom on Wednesday at 16h ?

**Reply by danielsgranja** [reply]
I have class at that time ^^'

**Reply by munoztd0** [reply]
But you said "Wednesday after 16h" right above ?

**Reply by danielsgranja** [reply]
Wednesday after 16h [...] i have class ^^'

**Reply by munoztd0** [reply]
Oh alright, sorry I got it. So Wednesday at 14h good for both of you @faureunige ?

**Reply by danielsgranja** [reply]
yes!

**Reply by faureunige** [reply]
works for me too :) thanks

**Comment by ap1987-ap** [comment]
output file: R_Markdown.knit.md

! Misplaced \noalign.
\bottomrule ->\noalign 
                       {\ifnum 0=`}\fi \@aboverulesep =\aboverulesep \global...
l.156 \bottomrule

Fehler: LaTeX failed to compile R_Markdown.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See R_Markdown.log for more info.
Ausf�hrung angehalten

**Reply by munoztd0** [reply]
Could you push you Rmarkdown file and any other thing you are working on github so I can have a look at the issue ?

**Reply by ap1987-ap** [reply]
Should I put the whole markdown exercise on github under my name?

**Reply by munoztd0** [reply]
Yes, put everything you are working on

**Reply by munoztd0** [reply]
I found the issue, it was with the "sumtable" function ! 
The default is html but if you want to print to pdf you just need to enter
`sumtable(iris, out="latex")`

**Comment by ap1987-ap** [comment]
Hi David

Thank you, I will try it as soon as I find the time.

Von: David Muñoz Tord ***@***.***>
Gesendet: Dienstag, 7. Dezember 2021 12:36
An: munoztd0/DAP_2021 ***@***.***>
Cc: Anja ***@***.***>; Mention ***@***.***>
Betreff: Re: [munoztd0/DAP_2021] Week 4 (Discussion #7)


I found the issue, it was with the "sumtable" function !
The default is html but if you want to print to pdf you just need to enter
sumtable(iris, out="latex")

**Comment by munoztd0** [comment]
I added a ["RESOURCES"](https://github.com/munoztd0/DAP_2021/tree/main/RESSOURCES) folder with a bunch of different example of code and [Markdown formatting](https://github.com/munoztd0/DAP_2021/blob/main/RESSOURCES/Rmardkown_tips_and_tricks.Rmd), try to look at it especially the ["multinomial_regression.R"](https://github.com/munoztd0/DAP_2021/blob/main/RESSOURCES/multinomial_regression.R) for @ap1987-ap  and the ["ordinal_regression.R"](https://github.com/munoztd0/DAP_2021/blob/main/RESSOURCES/ordinal_regression.R) for @danielsgranja .

**Reply by danielsgranja** [reply]
Thanks!

**Reply by ap1987-ap** [reply]
Hi @munoztd0 
I have a follow up question regarding multinomial regression. I looked up examples of its usage, and saw, that they evaluate the performance of the model using a test set. As we never did this in the lecture, should i still do it?
Best Anja

**Reply by munoztd0** [reply]
You should use as much stuff as possible that you understand and can explain. There will always be "better ways" and you can mention them, but stick with the stuff you are more comfortable with.

**Comment by ap1987-ap** [comment]
Ok, thank you.
I just wonder, because when I do the multinomial regression with all three groups, i get 65% accuracy, but when i try to do a glm with only two groups (e.g. PD and HC) I dont get any significant coefficients and a high AIC.

**Reply by munoztd0** [reply]
You cannot compare the two, different subset of data different question

**Comment by ap1987-ap** [comment]
Oh, ok, thank you. Ill try another approach.
